cmd/cmd-run.go

```diff
@@ -129,6 +131,10 @@ func run(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
+	if skipPullRequest && pushOnly {
+		return errors.New("--push-only and --skip-pr can't be used at the same time")
+	}
+
 	if skipPullRequest && forkMode {
 		return errors.New("--fork and --skip-pr can't be used at the same time")
 	}
```

Comment:
All of these are mutually exclusive.

Reply:
I have added another case to have forkMode and pushOnly conflict